viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -258,11 +257,11 @@ export class LighthouseReportViewer {
   // TODO: Really, `json` should really have type `unknown` and
   // we can have _validateReportJson verify that it's an LH.Result
   _replaceReportHtml(json) {
-    const container = dom.find('main', document);
+    const container = dom.find('main');
 
     // Reset container content.
     container.textContent = '';
-    const rootEl = document.createElement('div');
+    const rootEl = dom.createElement('div', 'lh-root lh-vars');
```

Comment:
`lh-root lh-vars` shouldn't be necessary for the flow report. Can you add these to the `rootEl` in `_renderLhr` instead?